dojo/tools/acunetix360/parser.py

```diff
@@ -51,6 +61,16 @@ def get_findings(self, filename, test):
                               cwe=cwe,
                               static_finding=True)
 
+            if item["State"].find("FalsePositive") != -1:
+                finding.active = False
+                finding.verified = False
+                finding.false_p = True
+                finding.mitigated = None
+                finding.is_mitigated = False
+
+            if item["State"].find("AcceptedRisk") != -1:
+                finding.risk_accepted = True
```

Comment:
I think that for this use case you should put `verified` at true and `active` to false

Reply:
updated verified and active fields too